### ISSUE TITLE
Video and Audio Call End (Button) not close call-screen

### ISSUE THREAD

**Issue by caglarturkurka** [cometchat-pro/javascript-chat-sdk#47]
Hi,  I use comet-chat-pro for my Angular 7.2.0 application. Everything works ver well but When I click end button console show errors and call-screen close too late. Please help me what is the problem ? I use comet-chat-pro => 1.8.13

**Comment by mayur-bhandari** [comment]
Hello @caglarturkurka , is your call screen not getting closed? We are looking into the error but the call screen should close on clicking end call button. Can you please let us know what issue are you facing exactly?

**Comment by caglarturkurka** [comment]
I got this error when click end button

[JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>: UnhandledError: null Script: null Line: null Column: null StackTrace: Error: Strophe: User connection callback caused an exception: SecurityError: Blocked a frame with origin "https://caesar.cometchat.com" from accessing a cross-origin frame.

Note: After I got this error, call screen is closed too late. Please help me what is the problem ? This error is written on chrome console.

**Comment by caglarturkurka** [comment]
Now I updated my  package.json of Angular Application from 7.2.0 to 8.2.11.But my  comet-chat-pro problem is still continue. Everything is working well with comet-chat-pro. But When I click end button for finishing Audio or Video Calling. Console writes lots of different error like

Error  => **[JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>: UnhandledError: null Script: null Line: null Column: null StackTrace: Error: Strophe: User connection callback caused an exception: SecurityError: Blocked a frame with origin "https://caesar.cometchat.com" from accessing a cross-origin frame.**


```
{
  "name": "example-frontend",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve --port 5000",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "prod": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build --prod --build-optimizer",
    "customer": "ng build --configuration=customer",
    "compile:server": "webpack --config webpack.server.config.js --progress --colors",
    "serve:ssr": "node local.js",
    "build:ssr": "npm run build:client-and-server-bundles && npm run compile:server",
    "build:client-and-server-bundles": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build --prod && ng run example-frontend:server:production",
    "server": "node local.js",
    "build:prod": "npm run build:browser:prod && npm run build:server:prod",
    "serve:prerender": "node static.js",
    "build:prerender": "npm run build:prod && node dist/prerender.js",
    "build:browser:prod": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build --prod",
    "build:browser:serverless": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build --prod --base-href /production/",
    "build:serverless": "npm run build:browser:serverless && npm run build:server:serverless",
    "build:prod:deploy": "npm run build:prod && npm run deploy",
    "build:serverless:deploy": "npm run build:serverless && npm run deploy",
    "deploy": "serverless deploy",
    "build:server:prod": "ng run example-frontend:server && webpack --config webpack.server.config.js --progress --colors",
    "build:server:serverless": "ng run example-frontend:server:serverless && webpack --config webpack.server.config.js --progress --colors"
  },
  "private": true,
  "dependencies": {
    "@agm/core": "^1.0.0-beta.5",
    "@angular/animations": "^8.2.11",
    "@angular/cdk": "^8.2.3",
    "@angular/common": "^8.2.11",
    "@angular/compiler": "^8.2.11",
    "@angular/core": "^8.2.11",
    "@angular/forms": "^8.2.11",
    "@angular/http": "^7.2.15",
    "@angular/platform-browser": "^8.2.11",
    "@angular/platform-browser-dynamic": "^8.2.11",
    "@angular/platform-server": "^8.2.11",
    "@angular/router": "^8.2.11",
    "@cometchat-pro/chat": "^1.8.13",
    "@ng-bootstrap/ng-bootstrap": "^4.0.1",
    "@ng-select/ng-select": "^2.16.2",
    "@ng-toolkit/serverless": "^7.1.2",
    "@ng-toolkit/universal": "^7.1.2",
    "@nguniversal/common": "0.0.0",
    "@nguniversal/express-engine": "0.0.0",
    "@nguniversal/module-map-ngfactory-loader": "0.0.0",
    "@ngx-translate/core": "^11.0.1",
    "@ngx-translate/http-loader": "^4.0.0",
    "@swimlane/ngx-charts": "^10.1.0",
    "@swimlane/ngx-datatable": "^15.0.1",
    "angular2-google-maps": "^0.17.0",
    "aws-serverless-express": "^3.2.0",
    "bootstrap": "^4.3.1",
    "classlist.js": "^1.1.20150312",
    "compass-mixins": "^0.12.10",
    "core-js": "^3.3.2",
    "cors": "~2.8.4",
    "cp-cli": "^1.1.0",
    "domino": "^2.1.3",
    "express": "^4.15.2",
    "font-awesome": "^4.7.0",
    "mock-browser": "^0.92.14",
    "moment": "^2.24.0",
    "ng-jhipster": "^0.5.6",
    "ng-recaptcha": "^4.2.1",
    "ng2-file-upload": "^1.3.0",
    "ng2-search-filter": "^0.4.7",
    "ng2-truncate": "^1.3.17",
    "ng5-slider": "^1.2.4",
    "ngx-bootstrap": "^5.2.0",
    "ngx-cookie-service": "^2.1.0",
    "ngx-embed-video": "^1.0.4",
    "ngx-hm-carousel": "^1.6.5-bata.1",
    "ngx-image-cropper": "2.0.0",
    "ngx-socket-io": "^3.0.1",
    "ngx-webstorage": "^3.0.0-beta.11",
    "ngx-xml2json": "^1.0.2",
    "rxjs": "^6.5.3",
    "sass": "^1.16.1",
    "socket.io-client": "^2.2.0",
    "terser": "3.14.1",
    "tslib": "^1.9.0",
    "typescript": "^3.5.3",
    "utf-8-validate": "^5.0.2",
    "web-animations-js": "^2.3.2",
    "zone.js": "^0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^0.803.12",
    "@angular/cli": "^8.3.12",
    "@angular/compiler-cli": "^8.2.11",
    "@angular/language-service": "^8.2.11",
    "@types/jasmine": "^3.4.4",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^12.11.1",
    "codelyzer": "^5.1.2",
    "jasmine-core": "^3.5.0",
    "jasmine-spec-reporter": "^4.2.1",
    "karma": "^4.4.1",
    "karma-chrome-launcher": "^3.1.0",
    "karma-cli": "^2.0.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "^2.0.1",
    "karma-jasmine-html-reporter": "^1.4.2",
    "opencollective": "^1.0.3",
    "protractor": "^5.4.2",
    "rxjs-tslint": "^0.1.7",
    "serverless": "1.40.0",
    "serverless-apigw-binary": "^0.4.4",
    "ts-loader": "^5.2.0",
    "ts-node": "~7.0.0",
    "tslint": "^5.20.0",
    "webpack": "^4.41.2",
    "webpack-cli": "^3.1.0"
  },
  "resolutions": {
    "terser": "3.14.1"
  }
}
```

**Comment by caglarturkurka** [comment]
Also I reviwed https://www.cometchat.com/tutorials/angular-voice-video-chat/ this example project and I upgraded comet-chat-pro version to 1.8.13 and same problem is occured in here. First time, end button works fine but If you try it more than one, browser gives this error => ` [JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>: UnhandledError: null Script: null Line: null Column: null StackTrace: Error: Strophe: User connection callback caused an exception: SecurityError: Blocked a frame with origin "https://caesar.cometchat.com" from accessing a cross-origin frame.`

**Comment by mayur-bhandari** [comment]
Hello @caglarturkurka , we have released new SDK version 2.0.0-beta9. Can you please check and let us know if this issue is solved in the latest release?

**Comment by mayur-bhandari** [comment]
Please follow this issue here #48 .
Closing due to duplicate issues.
